dir.targets

```diff
@@ -36,7 +36,7 @@
     <!-- Restore build tools -->
     <Exec
       StandardOutputImportance="Low"
-      Command="&quot;$(NuGetToolPath)&quot; install &quot; $(SourceDir).nuget\packages.config &quot;  -o &quot; $(ToolsDir) &quot; $(NuGetConfigCommandLine)" />
+      Command="&quot;$(NuGetToolPath)&quot; install &quot;$(PackagesGlobalConfig)&quot;  -o &quot; $(ToolsDir) &quot; $(NuGetConfigCommandLine)" />
```

Comment:
The spaces around `$(ToolsDir)` are still an issue (they will be taken literally on Linux, which means the folder will be invalid). The better fix is to use `&quot;$(ToolsDir.TrimEnd('\'))&quot;`